e2e_playwright/st_tabs_test.py

```diff
@@ -22,6 +23,9 @@ def test_tabs_render_correctly(themed_app: Page, assert_snapshot: ImageCompareFu
     st_tabs = themed_app.get_by_test_id("stTabs")
     expect(st_tabs).to_have_count(3)
 
+    st_tabs.nth(2).get_by_text("Tab 1", exact=True).click()
+    st_tabs.nth(2).get_by_text("Tab 0", exact=True).click()
```

Comment:
I would have expected these lines to modify the snapshots, but I don't see any updated snapshot files. Are these clicks no-ops?

Reply:
So, these lines were so that it *didn't* modify the snapshots.. Might not be needed if I can move the block code into map.